### PR TITLE
Remove streamer from whisper sample

### PR DESCRIPTION
**Pull request by as-suvorov**

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@sgolebiewski-intel

File: docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst [hunk 1]
```diff
@@ -249,23 +249,21 @@ and use audio files in WAV format at a sampling rate of 16 kHz as input.
...
-             raw_speech = read_wav(wav_file_path)
-             pipe = openvino_genai.WhisperPipeline(model_dir)
-
-             def streamer(word: str) -> bool:
-                 print(word, end="")
-                 return False
+             device = "CPU"  # GPU, NPU can be used as well
+             pipe = openvino_genai.WhisperPipeline(model_dir, device)
 
+             # Pipeline expects normalized audio with Sample Rate of 16kHz
```
Comment (sgolebiewski-intel) [inline docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst:255]:
```suggestion
             # The pipeline expects normalized audio with a sampling rate of 16kHz.
```

File: docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst [hunk 2]
```diff
@@ -288,26 +286,21 @@ and use audio files in WAV format at a sampling rate of 16 kHz as input.
...
              config.task = "transcribe";
              config.return_timestamps = true;
 
-             auto streamer = [](std::string word) {
-                 std::cout << word;
-                 return false;
-             };
-
-             auto result = pipeline.generate(raw_speech, config, streamer);
+             // Pipeline expects normalized audio with Sample Rate of 16kHz
```
Comment (sgolebiewski-intel) [inline docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst:299]:
```suggestion
             // The pipeline expects normalized audio with a sampling rate of 16kHz.
```

File: docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst [hunk 3]
```diff
@@ -249,23 +249,21 @@ and use audio files in WAV format at a sampling rate of 16 kHz as input.
 
 
          def infer(model_dir: str, wav_file_path: str):
-             raw_speech = read_wav(wav_file_path)
-             pipe = openvino_genai.WhisperPipeline(model_dir)
-
-             def streamer(word: str) -> bool:
-                 print(word, end="")
-                 return False
+             device = "CPU"  # GPU, NPU can be used as well
```
Comment (sgolebiewski-intel) [inline docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst:252]:
```suggestion
             device = "CPU"  # GPU or NPU can be used as well.
```

File: docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst [hunk 4]
```diff
@@ -288,26 +286,21 @@ and use audio files in WAV format at a sampling rate of 16 kHz as input.
 
              std::filesystem::path models_path = argv[1];
              std::string wav_file_path = argv[2];
-             std::string device = "CPU"; // GPU can be used as well
+             std::string device = "CPU";  // GPU, NPU can be used as well
```
Comment (sgolebiewski-intel) [inline docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst:289]:
```suggestion
             std::string device = "CPU";  // GPU or NPU can be used as well.
```

**Review by sgolebiewski-intel — APPROVED** [review]

**Pull request merged**